plugins/forc-index/src/commands/deploy.rs

```diff
@@ -6,8 +6,8 @@ use clap::Parser;
 #[derive(Debug, Parser)]
 pub struct Command {
     /// URL at which to upload index assets
-    #[clap(long, default_value = defaults::INDEXER_SERVICE_URL, help = "URL at which to upload index assets.")]
-    pub url: String,
+    #[clap(long, default_value = defaults::INDEXER_SERVICE_HOST, help = "URL at which to upload index assets.")]
+    pub host: String,
```

Comment:
@hrryndrsn 
- Doh! Given [this comment](https://github.com/FuelLabs/fuel-indexer/pull/486#discussion_r1081428753) this needs to be changed back to `url` for consistency 
- No changelog notes in this case (after I change it back). 
  - But yes, in general, if we change something in the plugin commands we'd want to make sure they're in the **Changelog** notes in the PR description so that they can be put in the release notes.